waiter/test/waiter/token_test.clj

```diff
@@ -58,29 +58,29 @@
           service-description2 (clojure.walk/stringify-keys
                                  {:cmd "tc2", :cpus 2, :mem 400, :version "d1e2f3", :run-as-user "tu1", :permitted-user "tu3", :token token})
           service-id2 (sd/service-description->service-id service-id-prefix service-description2)
-          waiter-hostname "waiter-hostname.app.example.com"
+          waiter-hostnames #{"waiter-hostname.app.example.com"}
           handle-list-tokens-request (wrap-handler-json-response handle-list-tokens-request)]
 
       (testing "put:unsupported-request-method"
         (let [{:keys [status body]}
               (run-handle-token-request
```

Comment:
`run-handle-token-request` parameter should be renamed to `waiter-hostnames` from `waiter-hostname` in the definition above.

Reply:
Done